### DISCUSSION TITLE
Ayuda con los wireframe

### DISCUSSION THREAD

**Discussion by ArmandoRiveraT** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#173] [Q&A]
Hola amigos espero que estén bien. Quiero pedirles de su ayuda para realizar mi UI y UX. Lo que pasa es que veo los UI que han realizado los demás y se ven muy bonitos con iconos y figuritas. Según tengo entendido puedo hacer eso en Miro, lo que pasa es que a mi no me aparecen nada de iconos ni diseños para usar en el UI :c Ando muy perdido en estos temas no sé usar Miro, ni AdobeXD y no se como hacer mi UI, me podrían ayudar por favor.

**Comment by JuanSalvador73** [comment]
De hecho, puedes usar cualquier programa gráfico que te sirve, yo uso photoshop e Illustrator, si bien los softwares listados en la capacitación son los ideales por que ya te facilitan todo. De Miro, checa este tutorial:

https://www.youtube.com/watch?v=BRuNwXAGEG8

**Reply by ArmandoRiveraT** [reply]
Muchas gracias amigo 🙏 tu video me esta siendo de mucha ayuda.

**Comment by DynamiteFang** [comment]
De los programas que nos pasó el MC yo usé Figma, no me quedó espectacular pero puedo decir que es útil y sólo es de agarrarle práctica y experimentar con las herramientas.

**Reply by ArmandoRiveraT** [reply]
Muchas gracias por responder amigo, le echaré un ojo a Figma a ver que tal.

**Comment by linettelopb** [comment]
Te recomiendo utilizes figma tiene multiples plugins que puedes utilizar :)

**Reply by ArmandoRiveraT** [reply]
Hola muchas gracias por la recomendación, le echaré un vistazo a Figma a ver que tal.
